src/common/components/Header/Header.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+import { Container } from 'common/components/Container/index.js'
```

Comment:
`index.js` files are special, because they're the default file for a folder. So, you can simplify this path to just `common/components/Container`.

Going further, if you import (and then export) all components inside of `components/index.js`, you can just do:

```import { Container } from 'common/components'```